third_party/googleapis/google/api/annotations.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015, Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.api;
+
+import "google/api/http.proto";
```

Comment:
I think that changing the import path is perhaps incorrect: `google/api/http.proto` seems to be the canonical import path as [files like these](https://github.com/google/googleapis/blob/master/google/api/annotations.proto#L19-L20) in the official Google repository import it that way.

> It also seems to be more in spirit of Go to use a full import path rather than a relative one.

In Go I would certainly agree, but with `protoc` I don't think there is any such preference.